src/tests/viewMode.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
```

Comment:
lets these tests in `App.test.tsx` instead so its easy to relate the test files with source files.

Reply:
I wouldn't connect source files to test files. 1) we're currently not really doing that 2) pretty much everything would be in `App.test.tsx`

Reply:
yes, it is sort of inconsistent right now and we are doing it only for unit tests, package/excalidraw tests but the reason I suggested doing it is
1. it's easy to find and relate the test files if we map it to the source files instead of having different tests files for certain features.
2. maintain consistency as you don't have to think whether a new file needs to be created.
3. having all tests related to App.tsx shouldn't be a problem as we will have a describe block for every new kind of test

Reply:
(1) Most features are strewn across multiple files, often across 10 or more. You can't make a clear distinction where a feature belongs to. Also, we'd have to constantly keep shuffling test code around files based on how the source code is refactored (it's much less likely a feature nomenclature is changed).
(2) For the reason above, I'd argue you'll spend way more time trying to figure out what test file to use.
(3) Huge files aren't good. Furthermore, if we use snapshots then the flow of using `.only` on specific test cases when developing/debugging is a nightmare in huge test files.

Reply:
Yeah I agree huge files aren't good of course (same goes for source code as well, 😉  test files are likely to be larger than source)
And since we are doing integration test mostly so that will cover code across multiple files in which case it might not make sense to map with a single file so we can keep diff files for integration tests feature wise, my worry here is often tests for a feature might spread across multiple files in case we don't realize there is a already a file for this feature or something so that's what we would need to take care of.

Reply:
>  my worry here is often tests for a feature might spread across multiple files in case we don't realize there is a already a file for this feature or something so that's what we would need to take care of.

Yeah, I hear you — we can think of something if it becomes a problem.